sdk/eventhub/azure-eventhubs/azure/eventhub/aio/client_async.py

```diff
@@ -85,33 +97,31 @@ def _create_auth(self, username=None, password=None):
                                                 get_jwt_token, http_proxy=http_proxy,
                                                 transport_type=transport_type)
 
+    async def _handle_exception(self, exception, retry_count, max_retries):
+        await _handle_exception(exception, retry_count, max_retries, self, log)
+
+    async def _close_connection(self):
+        self._conn_manager.reset_connection_if_broken()
+
     async def _management_request(self, mgmt_msg, op_type):
-        alt_creds = {
-            "username": self._auth_config.get("iot_username"),
-            "password": self._auth_config.get("iot_password")}
-        connect_count = 0
+        max_retries = self.config.max_retries
+        retry_count = 0
         while True:
-            connect_count += 1
-            mgmt_auth = self._create_auth(**alt_creds)
+            mgmt_auth = self._create_auth()
             mgmt_client = AMQPClientAsync(self.mgmt_target, auth=mgmt_auth, debug=self.config.network_tracing)
             try:
-                await mgmt_client.open_async()
+                conn = await self._conn_manager.get_connection(self.host, mgmt_auth)
```

Comment:
Currently the "SeparateConnectionManager" returns `None` here... what happens in that case?

Reply:
returns None means no shared connection. So separate connection is created.
This None will be passed to the .open(connection=conn) of AMQPClient, SendClient and ReceiveClient of uamqp.